Makefile

```diff
@@ -0,0 +1,42 @@
+FMT_OPTS = -co -XTypeApplications -o -XImportQualifiedPost
+FIND_OPTS = src test -type f -name '*.hs'
+
+build:
+	cabal build
+
+test:
+	cabal test
+
+# Dogfood the dev version
+analyze:
+	cabal run fossa -- analyze --output --debug --filter 'cabal@./'
+
+# Copy the built binary into the local root
+fossa:
+	cp $(shell cabal list-bin fossa) ./fossa
+
+install-local: fossa
+
+# Run analysis on the sandbox directory
+sandbox: fossa
+	./fossa analyze --output --debug --record ./sandbox > fossa.json
+
+# Run the sandbox with replay mode on
+replay: fossa fossa.debug.json
+	./fossa analyze --output --debug --replay fossa.debug.json ./sandbox > fossa.replay.json
+
+# Format everything (don't use this, use your IDE to format)
+# `@command` does not echo the command before running
+fmt:
+	@fourmolu --mode inplace ${FMT_OPTS} $(shell find ${FIND_OPTS})
+
+# Confirm everything is formatted without changing anything
+fmt-ck:
+	@fourmolu --mode check ${FMT_OPTS} $(shell find ${FIND_OPTS})
+	@echo "No formatting errors found"
+
+# Lint everything (This parses wrong sometimes, use your editor instead)
```

Comment:
> This parses wrong sometimes

That's alarming. When does this happen?

Reply:
When extensions are not activated.  For now it seems to be OK, and it helps that we don't use default extensions in cabal unless they're no-brainers.  But it can happen, and HLS prevents it.

Reply:
We can configure hlint extensions via flags or `.hlint.yaml` ([doc](https://github.com/ndmitchell/hlint#language-extensions)). Let's do that instead of leaving this spooky comment in.

Reply:
Before you merge - did you add these flags @scruffystuffs? I'm not seeing an `.hlint.yaml` either.

Reply:
That's already in the repo from an earlier PR.  See #258